source/extensions/transport_sockets/proxy_protocol/proxy_protocol.cc

```diff
@@ -21,17 +21,17 @@ namespace Extensions {
 namespace TransportSockets {
 namespace ProxyProtocol {
 
-UpstreamProxyProtocolStats generateUpstreamProxyProtocolStats(Stats::Scope& stats_scope) {
-  const char prefix[]{"upstream.proxyprotocol."};
+UpstreamProxyProtocolStats generateUpstreamProxyProtocolStats(Stats::Scope& stats_scope,
+                                                              absl::string_view prefix) {
```

Comment:
nit: I'm not sure this change is needed as there is only one call site so we don't need to vary the prefix string.

But this looks good to me anyway!

Reply:
done.